encoders/ffmpeg_test.go

```diff
@@ -140,7 +140,7 @@ var _ = Describe("FFmpeg Encoder", func() {
 					},
 				},
 				Status:           types.JobCreated,
-				Details:          "0%",
```

Comment:
I agree with you that we should add tests to cover this case, but maybe find another way to throw the error, because we might fix the problem that it cannot encode videos with no audio. Any suggestion?